### PR TITLE
Add unittest example to std.string.wrap

### PR DESCRIPTION
**Pull request by JackStouffer**
* added unittest example
* removed unnecessary function annotations

### CODE REVIEW

File: std/string.d [hunk 1]
```diff
@@ -6404,6 +6404,14 @@ S wrap(S)(S s, in size_t columns = 80, S firstindent = null,
     return result;
 }
 
+///
+@safe pure unittest
+{
+    assert(wrap("a short string", 7) == "a short\nstring\n");
+    assert(wrap("a short string", 7, "\t") == "\ta\nshort\nstring\n");
+    assert(wrap("a short string", 7, "\t", "    ") == "\ta\n    short\n    string\n");
```
Comment (burner) [inline std/string.d]: could you add an example the the column break is inside a word.

Comment (JackStouffer) [inline std/string.d]: Looking at the source, I think `wrap` is designed never to break inside a word.

Comment (burner) [inline std/string.d]: that is absolutely my point. If I have to look at the source what good is the example?

Comment (JackStouffer) [inline std/string.d]: Oh, ok. Miscommunication.

**Comment by JackStouffer** [comment]
@burner fixed.

File: std/string.d [hunk 2]
```diff
@@ -6404,6 +6404,18 @@ S wrap(S)(S s, in size_t columns = 80, S firstindent = null,
     return result;
 }
 
+///
+@safe pure unittest
+{
+    assert(wrap("a short string", 7) == "a short\nstring\n");
+
+    // wrap will not break inside of a word
```
Comment (burner) [inline std/string.d]: ~ " but at the next space"

**Comment by JackStouffer** [comment]
@burner fixed again

**Comment by burner** [comment]
LGTM

**Comment by burner** [comment]
Auto-merge toggled on

**Comment by JackStouffer** [comment]
Thanks!

**Pull request merged**